src/main/java/com/google/sps/tests/TestServlet.java

```diff
@@ -0,0 +1,67 @@
+package testing.servlets;
```

Comment:
Why is this Servlet being copied to this directory? It seems like it already exists at https://github.com/abundis28/mintern/blob/master/tests/java-db-test/src/main/java/testing/servlets/TestServlet.java